src/components/visualizations/Visualization.module.scss

```diff
@@ -4,7 +4,7 @@
     font-size: 1em;
 
     border: 0.2em solid orange;
-    margin: 1em;
+    margin: 0.9em;
```

Comment:
Hva tenker du her? Bare at det ser bedre ut i påvente av mitt redesign? eller er det noe jeg bør ta hensyn til

Reply:
Nei, egentlig bare at det så litt bedre ut nå. Var litt vel stor margin der, som gjorde at innholdet fikk akkurat for liten plass i forhold til designet til UDIR. Men vi sjekker jo alt slikt nå du er klar, men var jo greit inntil videre